app/models/petition.rb

```diff
@@ -250,6 +257,10 @@ def scheduled_for_debate
     end
   end
 
+  def notes
+    note.details if note
+  end
+
```

Comment:
I think at some point we wanted to add multiple notes, e.g. `has_many` instead of `has_one` - adding this shortcut may make it trickier so I'm not sure it's worth it.